bundles/org.openhab.binding.sonos/pom.xml

```diff
@@ -14,4 +14,13 @@
 
   <name>openHAB Add-ons :: Bundles :: Sonos Binding</name>
 
+  <dependencies>
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-inline</artifactId>
+      <version>3.6.28</version>
+      <scope>test</scope>
+    </dependency>
+  </dependencies>
```

Comment:
We should not use **mockito-inline**, please see https://github.com/openhab/openhab-addons/pull/12061#issuecomment-1016226346.

Reply:
How can I mock a static method without that (HttpUtil.executeUrl)?

Reply:
I will rather move the HTTP code outside the method I want to test.

Reply:
Changed.

Reply:
Even better.